public/app/features/explore/Logs/LogsTableNavField.tsx

```diff
@@ -0,0 +1,80 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { Checkbox, Icon, useTheme2 } from '@grafana/ui';
+
+import { fieldNameMeta } from './LogsTableWrap';
+
+function getStyles(theme: GrafanaTheme2) {
+  return {
+    dragIcon: css({
+      cursor: 'drag',
+      marginLeft: theme.spacing(1),
+      opacity: 0.4,
+    }),
+    labelCount: css({
+      marginLeft: theme.spacing(0.5),
+      marginRight: theme.spacing(0.5),
+      appearance: 'none',
+      background: 'none',
+      border: 'none',
+      fontSize: theme.typography.pxToRem(11),
+      opacity: 0.6,
+    }),
+    contentWrap: css({
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'space-between',
+      width: '100%',
+    }),
+    // Making the checkbox sticky and label scrollable for labels that are wider then the container
```

Comment:
I think I couldn.t get the "scrollable" label to work. Did it work for you?
![image](https://github.com/grafana/grafana/assets/8092184/d26a8e4e-ba2a-4cf8-9b33-325ed8f1f1ae)

Reply:
No this is another stale comment, I had previously added the ability to scroll oversized values in the initial implementations but it's not a pattern used commonly in Grafana so it was removed.